### ISSUE TITLE
oetl import does not rename mysql fields

### ISSUE THREAD

**Issue by on-meetsys** [orientechnologies/orientdb#7307]
## Expected behavior  
I am importing tables from mysql to orientdb thanks to oetl.sh.
I use the following setup :
```
{
  "config": {
    "log": "debug"
  },
  "extractor" : {
    "jdbc": { "driver": "com.mysql.jdbc.Driver",
              "url": "jdbc:mysql://localhost/DBFROM",
              "userName": "mysqluser",
              "userPassword": "mysqlpwd",
              "query": "SELECT m.firstname, m.name as lastname FROM member m" }
  },
  "transformers" : [
    {"vertex": { "class":"Member"}}
  ],
  "loader" : {
    "orientdb": {
      "dbType": "graph",
      "dbURL": "remote:localhost/mydb",
      "dbUser": "admin",
      "dbPassword": "admin",
      "serverUser": "root",
      "serverPassword": "root",
      "standardElementConstraints": false,
      "dbAutoCreate": true
    }
  }
}
```

## Actual behavior  

In orientdb studio, I query members :
`SELECT FROM Member`

the result returns members with {firstname, name} fields and not {firstname,lastname}

If I look at Member class, the fields are well {firstname,lastname} though.

It looks like the fields are not the same in the class and in the data. 

## Steps to reproduce  
see before

**Comment by robfrank** [comment]
Can you try to import on a clean db WITHOUT aliases?
```sql
SELECT firstname, name  FROM member

```
Thanks

**Comment by on-meetsys** [comment]
As expected, this works perfectly : the @class and the SELECT results both have {firstname,name}

This is the workaround I use to import my data, renaming name to lastname in the orientd database afterwards with :

    ALTER PROPERTY Member.name NAME lastname;
    UPDATE Member REMOVE name;

(btw, I wonder why I do have to remove the 'name' field, which is still present in data after the ALTER PROPERTY...)

**Comment by robfrank** [comment]
Discovered. The JDBC extractor was fetching the column name using getColumnName instead of getColumnLabel. The latter uses the alias, if present, or the column name, if not.
This fix will be available in the next 2.2.18
